### PR TITLE
add fail method

### PR DESCRIPTION
**Pull request by AdrieanKhisbe**
Implement #33 feature request

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -1978,6 +1978,43 @@ describe('expect()', () => {
...
+
+        let exception = false;
+        try {
+            Code.fail('Something wrong happened!');
+        }
+        catch (err) {
+            exception = err;
+        }
+
+        Hoek.assert(exception, exception);
```
Comment (cjihrig) [inline test/index.js]: This assertion can be dropped.

File: test/index.js [hunk 2]
```diff
@@ -1978,6 +1978,43 @@ describe('expect()', () => {
...
+        Hoek.assert(exception.message === 'Something wrong happened!', exception);
+        done();
+
```
Comment (cjihrig) [inline test/index.js]: Please remove this blank line

File: test/index.js [hunk 3]
```diff
@@ -1978,6 +1978,43 @@ describe('expect()', () => {
...
+        try {
+            Code.fail('Final Failure');
+            Code.fail('FAIL AGAIN');
+        }
+        catch (err) {
+            exception = err;
+        }
+
+        Hoek.assert(exception, exception);
+        Hoek.assert(exception.message === 'Final Failure', exception);
```
Comment (cjihrig) [inline test/index.js]: This is the only assertion you need. The other two can be removed.

File: test/index.js [hunk 4]
```diff
@@ -1978,6 +1978,43 @@ describe('expect()', () => {
...
+        Hoek.assert(exception.message !== 'FAIL AGAIN', exception);
+        done();
+    });
+
```
Comment (cjihrig) [inline test/index.js]: Please remove this blank line.

File: README.md [hunk 5]
```diff
@@ -633,6 +634,17 @@ var throws = function () {
...
 
+### `fail(message)`
+
+Make the test fail with `message`.
+
+```js
+var Code = require('code');
+var fail = Code.fail;
+
+fail('This should not occur');
```
Comment (cjihrig) [inline README.md]: Can you just call `Code.fail()` and drop the `var fail = Code.fail;`

**Comment by AdrieanKhisbe** [comment]
voila :)

**Comment by cjihrig** [comment]
Thanks!

**Comment by AdrieanKhisbe** [comment]
de rien :)

**Pull request merged**